test/unit/three_scale/policies/specification_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ThreeScale::Policies::SpecificationTest < ActiveSupport::TestCase
+
+  test '#valid? returns true when the schema is valid' do
+    doc = file_fixture('policies/apicast-policy.json').read
+    specification = ThreeScale::Policies::Specification.new(doc)
+
+    assert specification.valid?, 'specification should be valid'
+  end
+
+  test 'valid? returns false for an invalid specification and errors returns why' do
```

Comment:
Isn't this the same as https://github.com/3scale/porta/pull/593/commits/06e0039a695abf02fe116ad0f01ca54b3a4bfc54#diff-380a8b2b009abbc3d04c7128a0ec9c13R44?

Reply:
The one asserts that a `Policy` should be invalid if its specification is invalid. This one asserts if a `Policies::Specification` is invalid if the JSON is invalid.